setup.py

```diff
@@ -9,7 +9,7 @@
 
 setuptools.setup(
     name="tf-fastmri-data",
-    version="0.0.2",
+    version="0.2.0",
```

Comment:
ha it was this low haha...
then maybe
```suggestion
    version="0.0.3",
```